test/e2e/node_maintenance_test.go

```diff
@@ -35,224 +35,224 @@ var (
 
 var _ = Describe("Starting Maintenance", func() {
 
-		var controlPlaneNodes, workers []string
-		var controPlaneMaintenance *nmo.NodeMaintenance
+	var controlPlaneNodes, workers []string
```

Comment:
I don't think this file belongs into this PR

Reply:
It is moved to https://github.com/medik8s/node-maintenance-operator/pull/79